app/models/user.rb

```diff
@@ -5,6 +5,7 @@ class User < ApplicationRecord
   class UserAuthenticationException < StandardError; end
 
   belongs_to :organisation, optional: true
+  has_many :mou_signatures
```

Comment:
Is it strictly necessary to have the `has_many` associations here in `user` and `organisation`? I'm not sure I can see a use case right now.

Reply:
It's not necessary if users can't move between orgs. Currently they can if someone changes their organisation in the users screen.

I don't think we really have a use case where users have been around long enough to actually be in a position to sign an MOU in one department, move orgs and sign another one but its possible.

I went for this option here because it represents the db well and doesn't add much complexity. Is there a better way to structure the DB schema to not need it?

Reply:
I feel Toms, implementation makes sense. It allows ActiveRecord to be aware of the relationship and optimize for it.
so if you wanted to pull out the last signed MOU you could use `user.mou_signatures.last` without having to do a separate query to search it up for the user.